client/packages/android/app/src/main/AndroidManifest.xml

```diff
@@ -13,16 +24,18 @@
 
         </activity>
 
-        <provider android:name="androidx.core.content.FileProvider" android:authorities="${applicationId}.fileprovider" android:exported="false" android:grantUriPermissions="true">
-            <meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/file_paths"></meta-data>
+        <provider
+            android:name="androidx.core.content.FileProvider"
+            android:authorities="${applicationId}.fileprovider"
+            android:exported="false"
+            android:grantUriPermissions="true">
+            <meta-data
+                android:name="android.support.FILE_PROVIDER_PATHS"
+                android:resource="@xml/file_paths"></meta-data>
         </provider>
     </application>
 
     <!-- Permissions -->
 
     <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
Hmm does this mean we don't need camera and sd card permissions anymore ?

Reply:
strangely not. I can save the logs to the download folder, and can scan barcodes. Those are the only reasons for having file and camera permissions afaik.